AutomatedTesting/Gem/PythonTests/scripting/ScriptEvent_AddRemoveParameter_ActionsSuccessful.py

```diff
@@ -48,7 +48,7 @@ def ScriptEvent_AddRemoveParameter_ActionsSuccessful():
     qtpy_asset_editor = qtpy_o3de_editor.open_asset_editor()
 
     # 2) Create new Script Event Asset
-    qtpy_asset_editor.click_menu_bar_option("Script Events")
+    qtpy_asset_editor.click_menu_bar_option(SCRIPT_CANVAS_UI)
```

Comment:
`SCRIPT_CANVAS_UI` translates to `"Script Canvas"`

Shouldn't this be a const that references `"Script Events"`?

Reply:
yep! used the wrong constant by mistake since I was copy pasting in the same constant in the other test script